### PR TITLE
Translate readme to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by victormazevedo**
### Affected components 

- Docs

### CODE REVIEW

**Comment by shahidhk** [comment]
Thanks @victormazevedo! Instead of editing the README directly, will you add a new file in `translations` directory following the naming convention?

**Comment by victormazevedo** [comment]
Thank you, @shahidhk! Yes, I'm on it!

File: README.md [hunk 1]
```diff
@@ -214,4 +214,8 @@ This readme is available in the following translations:
 - [French :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
 - [Greek 🇬🇷](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
 
+<<<<<<< HEAD
```
Comment (marionschleifer) [inline README.md]: There are some unresolved conflicts here.

Comment (marionschleifer) [reply README.md]: Also, the Portuguese link doesn't show.

File: translations/README.portuguese.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+<a href="https://eepurl.com/dBUfJ5"><img src="https://img.shields.io/badge/newsletter-subscribe-yellow.svg?style=flat"></a>
+
+Hasura GraphQL Engine é um servidor extremamente rápido que oferece a você criar **instantâneamente, em tempo real, APIs GraphQL baseadas no Postgres**, com [**webhook triggers**](event-triggers.md) em eventos do banco de dados, e [**remote schemas**](remote-schemas.md) para a regra de negócio.
+
+Hasura te ajuda a contruir aplicativos GraphQL suportados pelo Postgres ou migrar uma aplicação já existente, que utiliza Postgres, para GraphQL.
+
+Leia mais em [hasura.io](https://hasura.io) e na [documentação](https://docs.hasura.io).
+
+------------------
+
```
Comment (marionschleifer) [inline translations/README.portuguese.md]: This doesn't show up correctly:

image

Comment (victormazevedo) [reply translations/README.portuguese.md]: Resolved show up problem!

File: translations/README.portuguese.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+Adicione lógica de negócios assíncrona acionada por eventos do banco de dados. Ideal para notificações, pipelines de dados Postgres ou processamento assíncrono - [leia mais](../event-triggers.md).
+
+### Dados derivados ou transformações de dados
+
+Transforme os dados em Postgres ou execute a regra de negócios para obter outro conjunto de dados que possa ser consultado usando o GraphQL Engine- [leia mais](https://docs.hasura.io/1.0/graphql/manual/queries/derived-data.html).
+
+## Demos
+
+Confira todos os exemplos da aplicação na pasta
+[community/examples](../community/examples).
```
[the remark this answers is not in the document]
Comment (sventech) [reply translations/README.portuguese.md]: should be `../community/sample-apps` ?

[the remark this answers is not in the document]
Comment (victormazevedo) [reply translations/README.portuguese.md]: Thanks @sventech! You're right!

File: README.md [hunk 4]
```diff
@@ -213,5 +213,6 @@ This readme is available in the following translations:
 - [Japanese :jp:](translations/README.japanese.md) (:pray: [@moksahero](https://github.com/moksahero))
 - [French :fr:](translations/README.french.md) (:pray: [@l0ck3](https://github.com/l0ck3))
 - [Greek 🇬🇷](translations/README.greek.md) (:pray: [@MIP2000](https://github.com/MIP2000))
+- [Portuguese 🇧🇷](translations/README..md) (:pray: [@victormazevedo](https://github.com/victormazevedo))
```
Comment (marionschleifer) [inline README.md]: This link is not working. Could you fix it please?

Comment (victormazevedo) [reply README.md]: Of course! Fixed it! :pray:

**Comment by marionschleifer** [comment]
@victormazevedo unfortunately, another contributor was quicker to implement the changes (we had 3 PRs for this translation), so we merged their PR. We're really sorry about this 😞 we appreciate your contribution and we'll of course send you the Hasura swag. Again, we're sorry 🙏

**Pull request closed without merging**